.cicd/Jenkinsfile

```diff
@@ -78,7 +78,9 @@ pipeline {
                 axes {
                     axis {
                         name 'SRW_PLATFORM'
-                        values 'cheyenne', 'gaea', 'hera', 'jet', 'orion' //, 'pclusternoaav2use1', 'azclusternoaav2eus1', 'gclusternoaav2usc1'
+                        // Uncomment line below to re-add use of Hera
+                        // values 'cheyenne', 'gaea', 'hera', 'jet', 'orion' //, 'pclusternoaav2use1', 'azclusternoaav2eus1', 'gclusternoaav2usc1'
+                        values 'cheyenne', 'gaea', 'jet', 'orion' //, 'pclusternoaav2use1', 'azclusternoaav2eus1', 'gclusternoaav2usc1'
```

Comment:
@MichaelLueken, just for clarification, why would we want to remove an option to run Jenkins tests on Hera?

Reply:
@JeffBeck-NOAA Previously, EPIC was allowed to use the nems account on Hera by EMC.  Unfortunately, the use of the account on Hera has been rescinded about a week ago.  When this happened, the platform team shutdown the Jenkins runner on Hera to ensure that no PRs are run on the machine.  The Jenkinsfile still attempts to submit jobs to Hera, which are forcefully aborted, leading to failure due to abort with the Jenkins tests.  It is best to temporarily comment out Hera to keep the tests from failing due to abort. Once EPIC acquires a new account for Hera, it will be easy to reactivate the Jenkins pipeline again.

Reply:
@JeffBeck-NOAA Simply put, EPIC is no longer able to run jobs on Hera.  EPIC has turned off the ability for Jenkins to run on Hera.  Since Hera is down in Jenkins, tasks that attempt to launch on Hera are forcefully aborted, leading to failures in the Jenkins tests (an example of this is @mkavulich's PR #600, the details for the `continuous-integration/jenkins/pr-merge` says `The build of this commit was aborted`).

This modification will ensure that jobs for Hera are no longer added to the build queue (as noted from the `continuous-integration/jenkins/pr-merge` entry passing in the PR).  Once EPIC has an account that it can use to launch jobs again, then the Jenkins runner on Hera will be restarted, and I will be able to uncomment out the Hera entries in the `.cicd/Jenkinsfile`.

Please let me know if you have any questions.

Reply:
Thanks, @MichaelLueken!  It's extremely unfortunate that EPIC no longer has the ability to run Jenkins on Hera, since it's such a critical machine to support.  Do you know why EPIC turned off access?  Does Jenkins use a specific project that is no longer available?

Reply:
@JeffBeck-NOAA I think the issue is that EPIC was utilizing EMC's nems account, but EMC decided that EPIC would no longer be able to use the nems account last week (it was quite surprising and all of EPIC is reeling from this decision).  Since Jenkins is maintained by EPIC, it is required to use EPIC's account on the various Tier-1 machines.  With the loss of the nems account, Jenkins is no longer allowed to run on Hera.  Once a new account for EPIC is set up on Hera, the Jenkins runner on Hera will be reactivated, the account will be updated, and then I will be able to restore Hera in the `.cicd/Jenkinsfile`.